styleguide/components/header/header.stories.mdx

```diff
@@ -30,21 +30,28 @@ import linksVariable from '!!raw-loader!./links_example.rb';
 
 The Haml partial the following locals:
```

Comment:
Worth updating this to mention that these locals are now within a `header` hash?

Reply:
have reverted to previous form as discussed